html/changelogs/example.yml

```diff
@@ -6,7 +6,7 @@
 # Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
 # When it is, any changes listed below will disappear.
 #
-# Valid Prefixes: 
```

Comment:
Unrelated to the PR, but I'll accept it

Reply:
sorry about that I've been noticing it for a while and just haven't wanted to make a pr to add one ".